docs/fastdds/xml_configuration/domainparticipant.rst

```diff
@@ -0,0 +1,397 @@
+|                                       | traffic to PDP only.                 |                     |                 |
++---------------------------------------+--------------------------------------+---------------------+-----------------+
+| ``<use_WriterLivelinessProtocol>``    | Indicates whether to use the |br|    | ``Boolean``         | :class:`true`   |
+|                                       | DataWriterLiveliness protocol.       |                     |                 |
++---------------------------------------+--------------------------------------+---------------------+-----------------+
+| ``<metatrafficUnicastLocatorList>``   | Metatraffic Unicast Locator List.    | |loclist|           |                 |
++---------------------------------------+--------------------------------------+---------------------+-----------------+
+| ``<metatrafficMulticastLocatorList>`` | Metatraffic Multicast Locator List.  | |loclist|           |                 |
++---------------------------------------+--------------------------------------+---------------------+-----------------+
+| ``<initialPeersList>``                | Initial peers.                       | |loclist|           |                 |
++---------------------------------------+--------------------------------------+---------------------+-----------------+
+| ``<DataReaderHistoryMemoryPolicy>``   | Memory policy for DataReaders. |br|  | |mempol|            | |mempoldefault| |
+|                                       | See :ref:`historyqospolicykind`.     |                     |                 |
+|                                       |                                      |                     |                 |
++---------------------------------------+--------------------------------------+---------------------+-----------------+
+| ``<DataWriterHistoryMemoryPolicy>``   | Memory policy for DataWriters. |br|  | |mempol|            | |mempoldefault| |
+|                                       | See :ref:`historyqospolicykind`.     |                     |                 |
++---------------------------------------+--------------------------------------+---------------------+-----------------+
+| ``<DataReaderPayloadSize>``           | Maximum payload size for |br|        | ``UInt32``          | 512             |
+|                                       | DataReaders.                         |                     |                 |
+|                                       |                                      |                     |                 |
++---------------------------------------+--------------------------------------+---------------------+-----------------+
+| ``<DataWriterPayloadSize>``           | Maximum payload size for |br|        | ``UInt32``          | 512             |
+|                                       | DataWriters.                         |                     |                 |
```

Comment:
What does this mean? Does it mean I need to set this to send something bigger than 512 kB? That is not the case